README.md

```diff
@@ -34,5 +36,9 @@ See the [contributing guide](CONTRIBUTING.md) for detailed instructions on how t
 
 ### Miscellaneous
 
+## Websites
```

Comment:
I'd consider `freecodecamp` to be more of a resource and we want to add these as links. Also, make sure you don't break the current structure when adding new categories.